Backend/.gitignore

```diff
@@ -0,0 +1,21 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
```

Comment:
Te pliki od STS są raczej niepotrzebne ( zweryfikujmy usuwając :D ), bo STS to jest odmiana eclipse.